bazel/BUILD

```diff
@@ -114,3 +158,86 @@ cc_binary(
     ],
     visibility = ["//visibility:public"],
 )
+
+load("@org_tensorflow//tensorflow:tensorflow.bzl", "tf_cc_binary")
+tf_cc_binary(
```

Comment:
Should we have an UT, where we `bazel build hello_tf` and then run the executable?
That keeps the "bazel building of executables" path under test in CI